## 65% of end-end tests have passed
```
Total executed tests: 51
Failed tests: 15
Expected to fail tests: 3
Passed tests: 33
```
```
IDs of failed tests: 727230,702732,704613,702957,703495,703297,727229,703496,703202,702947,702844,702807,702808,702775,702958 
```
```
IDs of expected to fail tests: 702948,727232,703629 
```
<h3>Failed tests (15)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=20179-collectibles-overview-page-fixes&PR_ID=20281&APK_NAME=20281.apk&TR_CASE_IDS=727230,702732,704613,702957,703495,703297,727229,703496,703202,702947,702844,702807,702808,702775,702958">Rerun failed tests</a></li><br/><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3582666">test_group_chat_pin_messages</a>, id: 702732 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_group_chat.py:19: in prepare_devices
    self.drivers, self.loop = create_shared_drivers(3)
base_test_case.py:328: in create_shared_drivers
    raise e
base_test_case.py:318: in create_shared_drivers
    test_suite_data.current_test.testruns[-1].jobs[drivers[i].session_id] = i + 1
 '_asyncio.Future' object has no attribute 'session_id'
```

<br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3582705">test_group_chat_mute_chat</a>, id: 703495 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_group_chat.py:19: in prepare_devices
    self.drivers, self.loop = create_shared_drivers(3)
base_test_case.py:328: in create_shared_drivers
    raise e
base_test_case.py:318: in create_shared_drivers
    test_suite_data.current_test.testruns[-1].jobs[drivers[i].session_id] = i + 1
 '_asyncio.Future' object has no attribute 'session_id' 
```

<b>[[Chat is not unmuted after expected time: https://github.com/status-im/status-mobile/issues/19627]]</b><br/><br/></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3582702">test_group_chat_send_image_save_and_share</a>, id: 703297 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_group_chat.py:19: in prepare_devices
    self.drivers, self.loop = create_shared_drivers(3)
base_test_case.py:328: in create_shared_drivers
    raise e
base_test_case.py:318: in create_shared_drivers
    test_suite_data.current_test.testruns[-1].jobs[drivers[i].session_id] = i + 1
 '_asyncio.Future' object has no attribute 'session_id'
```

<br/><br/></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3582701">test_group_chat_reactions</a>, id: 703202 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_group_chat.py:19: in prepare_devices
    self.drivers, self.loop = create_shared_drivers(3)
base_test_case.py:328: in create_shared_drivers
    raise e
base_test_case.py:318: in create_shared_drivers
    test_suite_data.current_test.testruns[-1].jobs[drivers[i].session_id] = i + 1
 '_asyncio.Future' object has no attribute 'session_id'
```

<br/><br/></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3582676">test_group_chat_join_send_text_messages_push</a>, id: 702807 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_group_chat.py:19: in prepare_devices
    self.drivers, self.loop = create_shared_drivers(3)
base_test_case.py:328: in create_shared_drivers
    raise e
base_test_case.py:318: in create_shared_drivers
    test_suite_data.current_test.testruns[-1].jobs[drivers[i].session_id] = i + 1
 '_asyncio.Future' object has no attribute 'session_id' 
```

<b>[[Issue with a message status - Sent instead of Delivered, https://github.com/status-im/status-mobile/issues/20126]]</b><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d0d20fd83ef741a891687f668dba7b6f?auth=ad723668df34b6802aff0a2501d0efce">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3582677">test_group_chat_offline_pn</a>, id: 702808 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_group_chat.py:19: in prepare_devices
    self.drivers, self.loop = create_shared_drivers(3)
base_test_case.py:328: in create_shared_drivers
    raise e
base_test_case.py:318: in create_shared_drivers
    test_suite_data.current_test.testruns[-1].jobs[drivers[i].session_id] = i + 1
 '_asyncio.Future' object has no attribute 'session_id'
```

<br/><br/></td></tr></tbody></table><h4>Class TestWalletMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3582712">test_wallet_send_asset_from_drawer</a>, id: 727230 </b></td></tr><tr><td><p><blockquote><div># STEP: Getting ETH amount in the wallet of the sender before transaction</div><div>Device 1: Find `WalletTab` by `accessibility id`: `wallet-stack-tab` </div></blockquote></p>

```
critical/test_wallet.py:151: in test_wallet_send_asset_from_drawer
    sender_balance, receiver_balance, eth_amount_sender, eth_amount_receiver = self._get_balances_before_tx()
critical/test_wallet.py:40: in _get_balances_before_tx
    self.wallet_1.wallet_tab.click()
../views/base_element.py:90: in click
    element = self.find_element()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 1: WalletTab by accessibility id: `wallet-stack-tab` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3582711">test_wallet_send_eth</a>, id: 727229 </b></td></tr><tr><td><p><blockquote><div>Device 1: Swiping right on element SlideButton </div><div>Device 1: Find `SlideButton` by `xpath`: `//*[@resource-id='slide-button-track']` </div></blockquote></p>

```
critical/test_wallet.py:131: in test_wallet_send_eth
    self.wallet_1.send_asset(address=self.receiver['address'], asset_name='Ether', amount=amount_to_send)
../views/wallet_view.py:130: in send_asset
    self.confirm_transaction()
../views/wallet_view.py:118: in confirm_transaction
    self.slide_and_confirm_with_password()
../views/wallet_view.py:112: in slide_and_confirm_with_password
    self.slide_button_track.slide()
../views/base_view.py:257: in slide
    self.swipe_right_on_element(width_percentage=1.3, start_x=100)
../views/base_element.py:308: in swipe_right_on_element
    location, size = self.get_element_coordinates()
../views/base_element.py:294: in get_element_coordinates
    element = self.find_element()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 1: SlideButton by xpath: `//*[@resource-id='slide-button-track']` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr></tbody></table><h4>Class TestDeepLinksOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3582709">test_links_open_universal_links_from_chat</a>, id: 704613 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `Text` by `accessibility id`: `community-title` </div><div>Device 1: `Text` is `Closed community` </div></blockquote></p>

```
critical/test_deep_and_universal_links.py:71: in test_links_open_universal_links_from_chat
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Community 'e2e community token-gated' was not requested to join by the url https://status.app/c/G0IAAGRy3S0pLm1NeQ91pwKgFaFGoIlkxXmDxITXXi3KdZ7vpb9-ERczhqwoFwMK6sCPnKwA#zQ3shtJ8A3rxUZWcfWDL3vKYQBt3jMPTuf44dJUGfjZjsek66
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ead72118952b4717bf3fcfe6882c7a24?auth=661d5c97bc95ebc965a7160a3ceec0d1#53">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3582670">test_links_deep_links</a>, id: 702775 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `BrowserTab` by `accessibility id`: `browser-stack-tab` </div><div>Device 1: Tap on found: BrowserTab </div></blockquote></p>

```
critical/test_deep_and_universal_links.py:115: in test_links_deep_links
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Community 'e2e community token-gated' was not requested to join by the deep link status.app://c/G0IAAGRy3S0pLm1NeQ91pwKgFaFGoIlkxXmDxITXXi3KdZ7vpb9-ERczhqwoFwMK6sCPnKwA#zQ3shtJ8A3rxUZWcfWDL3vKYQBt3jMPTuf44dJUGfjZjsek66
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ead72118952b4717bf3fcfe6882c7a24?auth=661d5c97bc95ebc965a7160a3ceec0d1#201">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUiTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3582706">test_1_1_chat_mute_chat</a>, id: 703496 </b></td></tr><tr><td><p><blockquote><div>Device 1: Click until `ChatMessageInput` by `accessibility id`: `chat-message-input` will be presented </div><div>Device 1: Looking for a message by text: after unmute </div></blockquote></p>

```
critical/chats/test_1_1_public_chats.py:664: in test_1_1_chat_mute_chat
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 New messages counter is not shown after unmute
E    Message text 'after unmute' is not shown in chat preview after unmute
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/463f71cd90b14bada607f6fcc8b1b388?auth=1757ac80600c2be1708f4267df1fed42#103">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/77a4763b39ff4c4b8a978fe1bedb10f2?auth=e9d5cb30580620c6a67f95748c0f2631#113">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePRTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3582696">test_activity_center_mentions</a>, id: 702957 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `Button` by `accessibility id`: `new-to-status-button` </div><div>Device 2: Find `Button` by `accessibility id`: `new-to-status-button` </div></blockquote></p>

```
Test setup failed: activity_center/test_activity_center.py:388: in prepare_devices
    self.loop.run_until_complete(
/usr/lib/python3.10/asyncio/base_events.py:649: in run_until_complete
    return future.result()
__init__.py:52: in run_in_parallel
    returns.append(await k)
/usr/lib/python3.10/concurrent/futures/thread.py:58: in run
    result = self.fn(*self.args, **self.kwargs)
../views/sign_in_view.py:238: in create_user
    self.i_m_new_in_status_button.click_until_presence_of_element(self.generate_keys_button)
../views/base_element.py:111: in click_until_presence_of_element
    el.click()
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webelement.py:94: in click
    self._execute(Command.CLICK_ELEMENT)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webelement.py:395: in _execute
    return self._parent.execute(command, params)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 The element 'By.accessibilityId: new-to-status-button' does not exist in DOM anymore; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#stale-element-reference-exception
E   Stacktrace:
E   io.appium.uiautomator2.common.exceptions.StaleElementReferenceException: The element 'By.accessibilityId: new-to-status-button' does not exist in DOM anymore
E   	at io.appium.uiautomator2.model.ElementsCache.restore(ElementsCache.java:117)
E   	at io.appium.uiautomator2.model.ElementsCache.get(ElementsCache.java:153)
E   	at io.appium.uiautomator2.handler.Click.safeHandle(Click.java:36)
E   	at io.appium.uiautomator2.handler.request.SafeRequestHandler.handle(SafeRequestHandler.java:59)
E   	at io.appium.uiautomator2.server.AppiumServlet.handleRequest(AppiumServlet.java:277)
E   	at io.appium.uiautomator2.server.AppiumServlet.handleHttpRequest(AppiumServlet.java:271)
E   	at io.appium.uiautomator2.http.ServerHandler.channelRead(ServerHandler.java:68)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)
E   	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:345)
E   	at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:102)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)
E   	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:345)
E   	at io.netty.channel.CombinedChannelDuplexHandler$DelegatingChannelHandlerContext.fireChannelRead(CombinedChannelDuplexHandler.java:435)
E   	at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:293)
E   	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:267)
E   	at io.netty.channel.CombinedChannelDuplexHandler.channelRead(CombinedChannelDuplexHandler.java:250)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)
E   	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:345)
E   	at io.netty.handler.timeout.IdleStateHandler.channelRead(IdleStateHandler.java:266)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)
E   	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:345)
E   	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1294)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)
E   	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:911)
E   	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:131)
E   	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:611)
E   	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:552)
E   	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:466)
E   	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:438)
E   	at io.netty.util.concurrent.SingleThreadEventExecutor$2.run(SingleThreadEventExecutor.java:140)
E   	at io.netty.util.concurrent.DefaultThreadFactory$DefaultRunnableDecorator.run(DefaultThreadFactory.java:144)
E   	at java.lang.Thread.run(Thread.java:1012)
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ee4da4826079403d840840b43fba5c90?auth=2a1728888d1198acf3b3f503c18d25ec">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4245810b83b3469787320b08da628261?auth=e35a7a01433af90781b874493b6fd750">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3582697">test_activity_center_admin_notification_accept_swipe</a>, id: 702958 </b></td></tr><tr><td>

```
Test setup failed: activity_center/test_activity_center.py:388: in prepare_devices
    self.loop.run_until_complete(
/usr/lib/python3.10/asyncio/base_events.py:649: in run_until_complete
    return future.result()
__init__.py:52: in run_in_parallel
    returns.append(await k)
/usr/lib/python3.10/concurrent/futures/thread.py:58: in run
    result = self.fn(*self.args, **self.kwargs)
../views/sign_in_view.py:238: in create_user
    self.i_m_new_in_status_button.click_until_presence_of_element(self.generate_keys_button)
../views/base_element.py:111: in click_until_presence_of_element
    el.click()
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webelement.py:94: in click
    self._execute(Command.CLICK_ELEMENT)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webelement.py:395: in _execute
    return self._parent.execute(command, params)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 The element 'By.accessibilityId: new-to-status-button' does not exist in DOM anymore; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#stale-element-reference-exception
E   Stacktrace:
E   io.appium.uiautomator2.common.exceptions.StaleElementReferenceException: The element 'By.accessibilityId: new-to-status-button' does not exist in DOM anymore
E   	at io.appium.uiautomator2.model.ElementsCache.restore(ElementsCache.java:117)
E   	at io.appium.uiautomator2.model.ElementsCache.get(ElementsCache.java:153)
E   	at io.appium.uiautomator2.handler.Click.safeHandle(Click.java:36)
E   	at io.appium.uiautomator2.handler.request.SafeRequestHandler.handle(SafeRequestHandler.java:59)
E   	at io.appium.uiautomator2.server.AppiumServlet.handleRequest(AppiumServlet.java:277)
E   	at io.appium.uiautomator2.server.AppiumServlet.handleHttpRequest(AppiumServlet.java:271)
E   	at io.appium.uiautomator2.http.ServerHandler.channelRead(ServerHandler.java:68)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)
E   	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:345)
E   	at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:102)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)
E   	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:345)
E   	at io.netty.channel.CombinedChannelDuplexHandler$DelegatingChannelHandlerContext.fireChannelRead(CombinedChannelDuplexHandler.java:435)
E   	at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:293)
E   	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:267)
E   	at io.netty.channel.CombinedChannelDuplexHandler.channelRead(CombinedChannelDuplexHandler.java:250)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)
E   	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:345)
E   	at io.netty.handler.timeout.IdleStateHandler.channelRead(IdleStateHandler.java:266)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)
E   	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:345)
E   	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1294)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)
E   	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:911)
E   	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:131)
E   	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:611)
E   	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:552)
E   	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:466)
E   	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:438)
E   	at io.netty.util.concurrent.SingleThreadEventExecutor$2.run(SingleThreadEventExecutor.java:140)
E   	at io.netty.util.concurrent.DefaultThreadFactory$DefaultRunnableDecorator.run(DefaultThreadFactory.java:144)
E   	at java.lang.Thread.run(Thread.java:1012)
```

<br/><br/></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3582685">test_community_links_with_previews_github_youtube_twitter_gif_send_enable</a>, id: 702844 </b></td></tr><tr><td><p><blockquote><div>Device 2: Wait for text element `EmojisNumber` to be equal to `1` </div><div>Device 2: Find `EmojisNumber` by `xpath`: `//*[starts-with(@text,'https://m.youtube.com/watch?v=Je7yErjEVt4')]/ancestor::android.view.ViewGroup[@content-desc='chat-item']/../..//*[@content-desc='emoji-reaction-4']/android.widget.TextView[2]` </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:643: in test_community_links_with_previews_github_youtube_twitter_gif_send_enable
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Link message reaction is not shown for the sender
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a80101ad88024c0a93c951f5d2ca5a92?auth=248c8163b112b2bc93f6412b922c6d1f#329">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0477a34b21fb4a5c9a537689a3f217d4?auth=d469a27688cf7d4ea7d5860ff87f7361#304">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3582694">test_activity_center_reply_read_unread_delete_filter_swipe</a>, id: 702947 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `Button` by `accessibility id`: `new-to-status-button` </div><div>Device 2: Click until `EditBox` by `accessibility id`: `profile-title-input` will be presented </div></blockquote></p>

```
Test setup failed: activity_center/test_activity_center.py:248: in prepare_devices
    self.loop.run_until_complete(
/usr/lib/python3.10/asyncio/base_events.py:649: in run_until_complete
    return future.result()
__init__.py:52: in run_in_parallel
    returns.append(await k)
/usr/lib/python3.10/concurrent/futures/thread.py:58: in run
    result = self.fn(*self.args, **self.kwargs)
../views/sign_in_view.py:238: in create_user
    self.i_m_new_in_status_button.click_until_presence_of_element(self.generate_keys_button)
../views/base_element.py:111: in click_until_presence_of_element
    el.click()
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webelement.py:94: in click
    self._execute(Command.CLICK_ELEMENT)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webelement.py:395: in _execute
    return self._parent.execute(command, params)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 The element 'By.accessibilityId: new-to-status-button' does not exist in DOM anymore; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#stale-element-reference-exception
E   Stacktrace:
E   io.appium.uiautomator2.common.exceptions.StaleElementReferenceException: The element 'By.accessibilityId: new-to-status-button' does not exist in DOM anymore
E   	at io.appium.uiautomator2.model.ElementsCache.restore(ElementsCache.java:117)
E   	at io.appium.uiautomator2.model.ElementsCache.get(ElementsCache.java:153)
E   	at io.appium.uiautomator2.handler.Click.safeHandle(Click.java:36)
E   	at io.appium.uiautomator2.handler.request.SafeRequestHandler.handle(SafeRequestHandler.java:59)
E   	at io.appium.uiautomator2.server.AppiumServlet.handleRequest(AppiumServlet.java:277)
E   	at io.appium.uiautomator2.server.AppiumServlet.handleHttpRequest(AppiumServlet.java:271)
E   	at io.appium.uiautomator2.http.ServerHandler.channelRead(ServerHandler.java:68)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)
E   	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:345)
E   	at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:102)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)
E   	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:345)
E   	at io.netty.channel.CombinedChannelDuplexHandler$DelegatingChannelHandlerContext.fireChannelRead(CombinedChannelDuplexHandler.java:435)
E   	at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:293)
E   	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:267)
E   	at io.netty.channel.CombinedChannelDuplexHandler.channelRead(CombinedChannelDuplexHandler.java:250)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)
E   	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:345)
E   	at io.netty.handler.timeout.IdleStateHandler.channelRead(IdleStateHandler.java:266)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)
E   	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:345)
E   	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1294)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)
E   	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:911)
E   	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:131)
E   	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:611)
E   	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:552)
E   	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:466)
E   	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:438)
E   	at io.netty.util.concurrent.SingleThreadEventExecutor$2.run(SingleThreadEventExecutor.java:140)
E   	at io.netty.util.concurrent.DefaultThreadFactory$DefaultRunnableDecorator.run(DefaultThreadFactory.java:144)
E   	at java.lang.Thread.run(Thread.java:1012)
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ab1a358a325c46c7a0da08dd8b687c3c?auth=39e02861c3728670c263242a426e64a1">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/31853402221d4ab9818108db64031e93?auth=2672fd445a601a284bc813571f44f900">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Expected to fail tests (3)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestCommunityMultipleDeviceMergedTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3582695">test_community_hashtag_links_to_community_channels</a>, id: 702948 </b></td></tr><tr><td><p><blockquote><div>Device 2: Find `Text` by `xpath`: `//android.view.ViewGroup[@content-desc='chat-item']//android.widget.TextView[contains(@text,'#cats')]` </div><div>Device 2: Looking for a message by text: it is just a message text </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:1124: in test_community_hashtag_links_to_community_channels
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Message in community channel is not visible for user before join 
```

<b>[[Can't enter channel after community is fetched for the first time, https://github.com/status-im/status-mobile/issues/20395]]</b><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6cb6bf8f53134285bfb7a7081799d69a?auth=91042a618828213f39e631258c0cc781#304">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2a4079094eff455c8791342f873d9f44?auth=b68ffd6962fffd327072dce83d4fff98#292">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3582708">test_community_join_when_node_owner_offline</a>, id: 703629 </b></td></tr><tr><td><p><blockquote><div>Device 2: Looking for community: 'open community' </div><div>Device 2: Click until `Text` by `accessibility id`: `community-description-text` will be presented </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:1212: in test_community_join_when_node_owner_offline
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Message in community channel is not visible for user before join, it was indicated as not sent for the sender before he went offline 
```

<b>[[Can't enter channel after community is fetched for the first time, https://github.com/status-im/status-mobile/issues/20395]]</b><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6cb6bf8f53134285bfb7a7081799d69a?auth=91042a618828213f39e631258c0cc781#493">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2a4079094eff455c8791342f873d9f44?auth=b68ffd6962fffd327072dce83d4fff98#468">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3582714">test_wallet_add_remove_watch_only_account</a>, id: 727232 </b></td></tr><tr><td><p><blockquote><div>Device 1: `Text` is `0x8d2413447ff297d30bdc475f6d5cb00254685aae` </div><div>Device 1: Click system back button </div></blockquote></p>

```
critical/test_wallet.py:255: in test_wallet_add_remove_watch_only_account
    self.home_view.driver.fail(
base_test_case.py:178: in fail
    pytest.fail('Device %s: %s' % (self.number, text))
 Device 1: Incorrect address '0x8d2413447ff297d30bdc475f6d5cb00254685aae' is shown when swiping between accounts, expected one is '0x8d2413447ff297d30bdc475f6d5cb00254685aae' 
```

<b>[[Missing networks in account address, https://github.com/status-im/status-mobile/issues/20166]]</b><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/aff38033ebec42e792323a934944c7c5?auth=e28549a0d834f128c36fd20eab3c21af#103">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Passed tests (33)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestCommunityMultipleDeviceMergedTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3582678">test_community_markdown_support</a>, id: 702809 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6cb6bf8f53134285bfb7a7081799d69a?auth=91042a618828213f39e631258c0cc781#185">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2a4079094eff455c8791342f873d9f44?auth=b68ffd6962fffd327072dce83d4fff98#168">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3582675">test_community_mentions_push_notification</a>, id: 702786 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6cb6bf8f53134285bfb7a7081799d69a?auth=91042a618828213f39e631258c0cc781#136">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2a4079094eff455c8791342f873d9f44?auth=b68ffd6962fffd327072dce83d4fff98#110">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3582686">test_community_leave</a>, id: 702845 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6cb6bf8f53134285bfb7a7081799d69a?auth=91042a618828213f39e631258c0cc781#301">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2a4079094eff455c8791342f873d9f44?auth=b68ffd6962fffd327072dce83d4fff98#270">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUiTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3582674">test_1_1_chat_delete_via_long_press_relogin</a>, id: 702784 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/463f71cd90b14bada607f6fcc8b1b388?auth=1757ac80600c2be1708f4267df1fed42#169">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/77a4763b39ff4c4b8a978fe1bedb10f2?auth=e9d5cb30580620c6a67f95748c0f2631#126">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3582673">test_1_1_chat_is_shown_message_sent_delivered_from_offline</a>, id: 702783 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/463f71cd90b14bada607f6fcc8b1b388?auth=1757ac80600c2be1708f4267df1fed42#92">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/77a4763b39ff4c4b8a978fe1bedb10f2?auth=e9d5cb30580620c6a67f95748c0f2631#85">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityCenterContactRequestMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3582671">test_add_contact_field_validation</a>, id: 702777 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2afe410610b24f65888ba65903a27fd4?auth=9933f68651c582dee85b2a8994e5fac4#135">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2d59805589f24d4ba859619b9de315c3?auth=e3400d017da22968ef07a03eb36f08d0#212">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3582689">test_activity_center_contact_request_accept_swipe_mark_all_as_read</a>, id: 702851 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2afe410610b24f65888ba65903a27fd4?auth=9933f68651c582dee85b2a8994e5fac4#100">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2d59805589f24d4ba859619b9de315c3?auth=e3400d017da22968ef07a03eb36f08d0#118">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3582688">test_activity_center_contact_request_decline</a>, id: 702850 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2afe410610b24f65888ba65903a27fd4?auth=9933f68651c582dee85b2a8994e5fac4#57">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2d59805589f24d4ba859619b9de315c3?auth=e3400d017da22968ef07a03eb36f08d0#56">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3582672">test_1_1_chat_emoji_send_reply_and_open_link</a>, id: 702782 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/341fb7fed6534ca88412e71659aefc74?auth=5719eacf41e9bebae34ff7a20c1a9a0f#148">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d0de9164045646c5a94405572e944a07?auth=64aa4ced4a80ca48962f2d6b77f9dfa8#120">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3582667">test_1_1_chat_text_message_delete_push_disappear</a>, id: 702733 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/341fb7fed6534ca88412e71659aefc74?auth=5719eacf41e9bebae34ff7a20c1a9a0f#563">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d0de9164045646c5a94405572e944a07?auth=64aa4ced4a80ca48962f2d6b77f9dfa8#401">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3582679">test_1_1_chat_push_emoji</a>, id: 702813 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/341fb7fed6534ca88412e71659aefc74?auth=5719eacf41e9bebae34ff7a20c1a9a0f#422">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d0de9164045646c5a94405572e944a07?auth=64aa4ced4a80ca48962f2d6b77f9dfa8#260">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3582669">test_1_1_chat_non_latin_messages_stack_update_profile_photo</a>, id: 702745 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/341fb7fed6534ca88412e71659aefc74?auth=5719eacf41e9bebae34ff7a20c1a9a0f#364">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d0de9164045646c5a94405572e944a07?auth=64aa4ced4a80ca48962f2d6b77f9dfa8#210">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3582690">test_1_1_chat_edit_message</a>, id: 702855 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/341fb7fed6534ca88412e71659aefc74?auth=5719eacf41e9bebae34ff7a20c1a9a0f#471">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d0de9164045646c5a94405572e944a07?auth=64aa4ced4a80ca48962f2d6b77f9dfa8#291">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3582704">test_1_1_chat_send_image_save_and_share</a>, id: 703391 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/341fb7fed6534ca88412e71659aefc74?auth=5719eacf41e9bebae34ff7a20c1a9a0f#502">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d0de9164045646c5a94405572e944a07?auth=64aa4ced4a80ca48962f2d6b77f9dfa8#341">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3582665">test_1_1_chat_pin_messages</a>, id: 702731 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/341fb7fed6534ca88412e71659aefc74?auth=5719eacf41e9bebae34ff7a20c1a9a0f#210">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d0de9164045646c5a94405572e944a07?auth=64aa4ced4a80ca48962f2d6b77f9dfa8#165">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3582664">test_1_1_chat_message_reaction</a>, id: 702730 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/341fb7fed6534ca88412e71659aefc74?auth=5719eacf41e9bebae34ff7a20c1a9a0f#96">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d0de9164045646c5a94405572e944a07?auth=64aa4ced4a80ca48962f2d6b77f9dfa8#85">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3582699">test_restore_multiaccount_with_waku_backup_remove_switch</a>, id: 703133 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7f4ccb77817b47cabaa5aadb9f18dc49?auth=35222c76305d5d17387728264136cc55#230">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3582668">test_community_copy_and_paste_message_in_chat_input</a>, id: 702742 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7f4ccb77817b47cabaa5aadb9f18dc49?auth=35222c76305d5d17387728264136cc55#77">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3582707">test_community_discovery</a>, id: 703503 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7f4ccb77817b47cabaa5aadb9f18dc49?auth=35222c76305d5d17387728264136cc55#355">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3582692">test_community_undo_delete_message</a>, id: 702869 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7f4ccb77817b47cabaa5aadb9f18dc49?auth=35222c76305d5d17387728264136cc55#106">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3582687">test_community_navigate_to_channel_when_relaunch</a>, id: 702846 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7f4ccb77817b47cabaa5aadb9f18dc49?auth=35222c76305d5d17387728264136cc55#53">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3582703">test_community_mute_community_and_channel</a>, id: 703382 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7f4ccb77817b47cabaa5aadb9f18dc49?auth=35222c76305d5d17387728264136cc55#125">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3582700">test_community_several_images_send_reply</a>, id: 703194 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a80101ad88024c0a93c951f5d2ca5a92?auth=248c8163b112b2bc93f6412b922c6d1f#232">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0477a34b21fb4a5c9a537689a3f217d4?auth=d469a27688cf7d4ea7d5860ff87f7361#166">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3582691">test_community_one_image_send_reply</a>, id: 702859 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a80101ad88024c0a93c951f5d2ca5a92?auth=248c8163b112b2bc93f6412b922c6d1f#257">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0477a34b21fb4a5c9a537689a3f217d4?auth=d469a27688cf7d4ea7d5860ff87f7361#227">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3582682">test_community_emoji_send_copy_paste_reply</a>, id: 702840 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a80101ad88024c0a93c951f5d2ca5a92?auth=248c8163b112b2bc93f6412b922c6d1f#305">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0477a34b21fb4a5c9a537689a3f217d4?auth=d469a27688cf7d4ea7d5860ff87f7361#287">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3582698">test_community_mark_all_messages_as_read</a>, id: 703086 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a80101ad88024c0a93c951f5d2ca5a92?auth=248c8163b112b2bc93f6412b922c6d1f#551">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0477a34b21fb4a5c9a537689a3f217d4?auth=d469a27688cf7d4ea7d5860ff87f7361#473">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3582693">test_community_contact_block_unblock_offline</a>, id: 702894 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a80101ad88024c0a93c951f5d2ca5a92?auth=248c8163b112b2bc93f6412b922c6d1f#424">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0477a34b21fb4a5c9a537689a3f217d4?auth=d469a27688cf7d4ea7d5860ff87f7361#395">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3582710">test_community_edit_delete_message_when_offline</a>, id: 704615 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a80101ad88024c0a93c951f5d2ca5a92?auth=248c8163b112b2bc93f6412b922c6d1f#586">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0477a34b21fb4a5c9a537689a3f217d4?auth=d469a27688cf7d4ea7d5860ff87f7361#499">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3582681">test_community_message_delete</a>, id: 702839 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a80101ad88024c0a93c951f5d2ca5a92?auth=248c8163b112b2bc93f6412b922c6d1f#201">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0477a34b21fb4a5c9a537689a3f217d4?auth=d469a27688cf7d4ea7d5860ff87f7361#137">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3582680">test_community_message_send_check_timestamps_sender_username</a>, id: 702838 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a80101ad88024c0a93c951f5d2ca5a92?auth=248c8163b112b2bc93f6412b922c6d1f#159">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0477a34b21fb4a5c9a537689a3f217d4?auth=d469a27688cf7d4ea7d5860ff87f7361#108">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>9. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3582684">test_community_message_edit</a>, id: 702843 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a80101ad88024c0a93c951f5d2ca5a92?auth=248c8163b112b2bc93f6412b922c6d1f#177">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0477a34b21fb4a5c9a537689a3f217d4?auth=d469a27688cf7d4ea7d5860ff87f7361#125">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>10. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3582683">test_community_unread_messages_badge</a>, id: 702841 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a80101ad88024c0a93c951f5d2ca5a92?auth=248c8163b112b2bc93f6412b922c6d1f#394">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0477a34b21fb4a5c9a537689a3f217d4?auth=d469a27688cf7d4ea7d5860ff87f7361#365">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3582713">test_wallet_add_remove_regular_account</a>, id: 727231 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/aff38033ebec42e792323a934944c7c5?auth=e28549a0d834f128c36fd20eab3c21af#39">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

--- a/src/quo/components/profile/expanded_collectible/style.cljs
+++ b/src/quo/components/profile/expanded_collectible/style.cljs
@@ -11,7 +11,9 @@
   {:width            "100%"
    :aspect-ratio     (if square? 1 aspect-ratio)
    :border-radius    16
-   :background-color (colors/theme-colors colors/white colors/neutral-95 theme)})
+   :background-color (colors/theme-colors colors/white colors/neutral-95 theme)
+   :border-width     1
+   :border-color     (colors/theme-colors colors/neutral-80-opa-5 colors/white-opa-5 theme)})
 
 (defn fallback
   [{:keys [theme]}]

--- a/src/quo/components/profile/expanded_collectible/style.cljs
+++ b/src/quo/components/profile/expanded_collectible/style.cljs
@@ -7,10 +7,11 @@
    :border-radius   16})
 
 (defn image
-  [square? aspect-ratio]
-  {:width         "100%"
-   :aspect-ratio  (if square? 1 aspect-ratio)
-   :border-radius 16})
+  [square? aspect-ratio theme]
+  {:width            "100%"
+   :aspect-ratio     (if square? 1 aspect-ratio)
+   :border-radius    16
+   :background-color (colors/theme-colors colors/white colors/neutral-95 theme)})
 
 (defn fallback
   [{:keys [theme]}]

--- a/src/quo/components/profile/expanded_collectible/view.cljs
+++ b/src/quo/components/profile/expanded_collectible/view.cljs
@@ -66,7 +66,7 @@
        (and (not image-error?) supported-file?)
        [rn/view
         [rn/image
-         {:style     (style/image square? (:aspect-ratio image-size))
+         {:style     (style/image square? (:aspect-ratio image-size) theme)
           :source    image-src
           :native-ID native-ID
           :on-error  #(set-image-error true)

--- a/src/quo/components/profile/expanded_collectible/view.cljs
+++ b/src/quo/components/profile/expanded_collectible/view.cljs
@@ -21,9 +21,9 @@
       :value           counter}]))
 
 (defn- fallback-view
-  [{:keys [label theme counter]}]
-  [rn/view
-   {:style (style/fallback {:theme theme})}
+  [{:keys [label theme counter on-mount]}]
+  (rn/use-mount on-mount)
+  [rn/view {:style (style/fallback {:theme theme})}
    [counter-view counter]
    [rn/view
     [icon/icon :i/sad {:color (colors/theme-colors colors/neutral-40 colors/neutral-50 theme)}]]
@@ -53,15 +53,17 @@
      (cond
        (not supported-file?)
        [fallback-view
-        {:label   (i18n/label :t/unsupported-file)
-         :counter counter
-         :theme   theme}]
+        {:label    (i18n/label :t/unsupported-file)
+         :counter  counter
+         :theme    theme
+         :on-mount on-collectible-load}]
 
        image-error?
        [fallback-view
-        {:label   (i18n/label :t/cant-fetch-info)
-         :counter counter
-         :theme   theme}]
+        {:label    (i18n/label :t/cant-fetch-info)
+         :counter  counter
+         :theme    theme
+         :on-mount on-collectible-load}]
 
        (and (not image-error?) supported-file?)
        [rn/view

--- a/src/status_im/contexts/wallet/collectible/view.cljs
+++ b/src/status_im/contexts/wallet/collectible/view.cljs
@@ -73,7 +73,9 @@
     :label               (i18n/label :t/about)
     :accessibility-label :about-tab}])
 
-(def navigate-back #(rf/dispatch [:navigate-back]))
+(defn navigate-back-and-clear-collectible []
+  (rf/dispatch [:navigate-back])
+  (rf/dispatch [:wallet/clear-last-collectible-details]))
 
 (defn animated-header
   [{:keys [scroll-amount title-opacity page-nav-type picture title description theme]}]
@@ -100,7 +102,7 @@
          :background          :blur
          :icon-name           :i/close
          :accessibility-label :back-button
-         :on-press            navigate-back
+         :on-press            navigate-back-and-clear-collectible
          :right-side          [{:icon-name :i/options
                                 :on-press  #(rf/dispatch
                                              [:show-bottom-sheet
@@ -242,9 +244,6 @@
         {preview-uri :uri}         preview-url
         {collectible-name :name}   collectible-data
         {collection-name :name}    collection-data]
-
-    (rn/use-unmount #(rf/dispatch [:wallet/clear-last-collectible-details]))
-
     [rn/view {:style (style/background-color theme)}
      [animated-header
       {:scroll-amount scroll-amount

